lib/clowne/adapters/sequel/associations/base.rb

```diff
@@ -1,21 +1,27 @@
 # frozen_string_literal: true
 
+require 'forwardable'
 require 'clowne/adapters/base/association'
 
 module Clowne
   module Adapters # :nodoc: all
     class Sequel
       module Associations
         class Base < Base::Association
+          extend Forwardable
+
           private
 
-          def clone_record(record)
-            Clowne::Adapters::Sequel::Copier.call(record)
-          end
+          def_delegators :operation, :record_wrapper
+          def_delegators Clowne::Adapters::Sequel, :dup_record
```

Comment:
What about adding a class-level instance variable `adapter_class` (in `Base::Association`) and set it here?

I mean:

```ruby
self.adapter_class = Clowne::Adapters::Sequel
```

That looks more clear to me than delegation.
And we can also use `adapter_class` in `#operation`.